src/core/event/event.proto

```
@@ -0,0 +1,33 @@
+syntax = "proto3";
+
+package com.eligius.event;
```

Buf (1.47.2)

Files with package "com.eligius.event" must be within a directory "com/eligius/event" relative to root but were in directory "src/core/event".

(PACKAGE_DIRECTORY_MATCH)